### PR TITLE
Updated links to tutorial and git repo

### PR DESCRIPTION
**Pull request by tristangoogle**

### CODE REVIEW

File: appengine/standard/firebase/firenotes/README.md [hunk 1]
```diff
@@ -45,7 +51,7 @@ server with the following command:
 
         dev_appserver.py frontend/app.yaml backend/app.yaml
 
-1. Visit [http://locahost:8080/](http://locahost:8080/) in a web browser.
+1. Visit <http://locahost:8080/> in a web browser.
```
Comment (jonparrott) [inline appengine/standard/firebase/firenotes/README.md]: Why change this? It works as a link the way you had it before?

Comment (tristangoogle) [inline appengine/standard/firebase/firenotes/README.md]: True, no reason except I learned it was possible to do.

Comment (jonparrott) [inline appengine/standard/firebase/firenotes/README.md]: I'd prefer the other way, for this and the link to the docs page.

**Comment by jonparrott** [comment]
This branch seems to have conflict, can you try to resolve those?

**Comment by tristangoogle** [comment]
Is it the requirements.txt file? I thought I updated that

**Comment by jonparrott** [comment]
I remembered I have access to your fork and just fixed it for you. Merged.

**Comment by tristangoogle** [comment]
Thank you!

**Pull request merged**